### PR TITLE
Adds 3 new tool abnormalities

### PR DESCRIPTION
**Pull request by Coxswain-Navigator**
## About The Pull Request
Adds Dr. Jekyll's formula, Viva Voce, and the Theonite slab. Each are different flavors of skin prophecy.

- Dr. Jekyll's formula increases fortitude with each use, while lowering RED resistance. Causes the user to take health damage from white damage and fear.
- Viva Voce increases temperance with each use, while lowering BLACK resistance. Causes the user to take double work damage.
- The Theonite Slab increases justice with every use, while lowering PALE resistance. Causes the user to be attacked with an AOE of PALE damage equal to damage taken when the user takes a large amount of damage. The AOE only damages people who have previously used the slab.

## Why It's Good For The Game
Is it? Some of these will probably get changed in the balance pass.

## Changelog
:cl:
add: Added new tool abnormalities
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/abnormality/_tools/he/theonite_slab.dm [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+/datum/status_effect/slab/proc/PunishHit(turf/R, damage, damagetype)
+	var/mob/living/carbon/human/H
+	for(var/turf/T in view(2, R))
+		new /obj/effect/temp_visual/smash_effect(T)
+		for(H in T)
+			var/datum/status_effect/slab/S = H.has_status_effect(/datum/status_effect/slab)
+			if(!S)
+				continue
+			H.apply_damage(damage, PALE_DAMAGE, null, H.run_armor_check(null, PALE_DAMAGE), spread_damage = TRUE)
+	playsound(H, 'sound/effects/ordeals/white/pale_teleport_in.ogg', 50, FALSE, 3)
```
Comment (EgorDinamit) [inline code/modules/mob/living/simple_animal/abnormality/_tools/he/theonite_slab.dm:65]:
```suggestion
	for(var/turf/T in view(2, R))
		new /obj/effect/temp_visual/smash_effect(T)
		for(var/mob/living/carbon/human/H in T)
			var/datum/status_effect/slab/S = H.has_status_effect(/datum/status_effect/slab)
			if(!S)
				continue
			H.apply_damage(damage, PALE_DAMAGE, null, H.run_armor_check(null, PALE_DAMAGE), spread_damage = TRUE)
	playsound(H, 'sound/effects/ordeals/white/pale_teleport_in.ogg', 35, TRUE)
```

File: code/modules/mob/living/simple_animal/abnormality/_tools/he/vivavoce.dm [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+	UnregisterSignal(owner, COMSIG_WORK_STARTED)
+	UnregisterSignal(owner, COMSIG_WORK_COMPLETED)
+	return ..()
+
+/datum/status_effect/rolecall/proc/UserDebuff()
+	SIGNAL_HANDLER
+	var/mob/living/carbon/human/H = owner
+	if(!istype(H))
+		return
+	to_chat(H, "<span class='userdanger'>The abnormality seems more aggressive than usual!</span>")
```
Comment (EgorDinamit) [inline code/modules/mob/living/simple_animal/abnormality/_tools/he/vivavoce.dm:49]: Might want to either remove it, or change the text to refer to tool itself.

File: code/modules/mob/living/simple_animal/abnormality/_tools/teth/dr_jekyll.dm [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+// Status Effect //TODO: make it spawn a hosite copy of you somehow.
+/datum/status_effect/dr_jekyll
+	id = "dr_jekyll"
+	status_type = STATUS_EFFECT_UNIQUE
+	duration = -1
+	alert_type = null
+
+/datum/status_effect/dr_jekyll/on_apply()
+	RegisterSignal(owner, COMSIG_MOB_APPLY_DAMGE, .proc/HydeDam)
+	RegisterSignal(owner, COMSIG_FEAR_EFFECT, .proc/HydeFear)
```
Comment (EgorDinamit) [inline code/modules/mob/living/simple_animal/abnormality/_tools/teth/dr_jekyll.dm:34]: Might want to combine the two: Add a signal for taking sanity damage in general.

Comment (Coxswain-Navigator) [reply code/modules/mob/living/simple_animal/abnormality/_tools/teth/dr_jekyll.dm:34]: Probably don't want to for this specific situation due to black damage not being intended to apply here.

File: code/modules/mob/living/simple_animal/abnormality/_tools/he/theonite_slab.dm [hunk 4]
```diff
@@ -0,0 +1,76 @@
+#define STATUS_EFFECT_SLAB /datum/status_effect/slab
+/obj/structure/toolabnormality/theonite_slab
+	name = "theonite slab"
+	desc = "A slab, made out of a seamless mixture of stone and metal. It's covered in runes, and bloody spikes erupt from the centerpiece."
+	icon_state = "slab"
+	var/list/users = list()
+
+/obj/structure/toolabnormality/theonite_slab/attack_hand(mob/living/carbon/human/user)
+	..()
+	if(!do_after(user, 6))
```
Comment (dsmith328) [inline code/modules/mob/living/simple_animal/abnormality/_tools/he/theonite_slab.dm:10]:
```suggestion
	if(!do_after(user, 6, src))
```
Stops the user spamming it for rapid stacks, despite being funny.

File: code/modules/mob/living/simple_animal/abnormality/_tools/he/vivavoce.dm [hunk 5]
```diff
@@ -0,0 +1,69 @@
+#define STATUS_EFFECT_ROLECALL /datum/status_effect/rolecall
+/obj/structure/toolabnormality/vivavoce
+	name = "viva voce"
+	desc = "A very old-fashioned communication device."
+	icon_state = "vivavoce"
+	var/list/users = list()
+
+/obj/structure/toolabnormality/vivavoce/attack_hand(mob/living/carbon/human/user)
+	..()
+	if(!do_after(user, 6))
```
Comment (dsmith328) [inline code/modules/mob/living/simple_animal/abnormality/_tools/he/vivavoce.dm:10]:
```suggestion
	if(!do_after(user, 6, src))
```
Stops it from being spammed, regardless of comedic effect.

File: code/modules/mob/living/simple_animal/abnormality/_tools/teth/dr_jekyll.dm [hunk 6]
```diff
@@ -0,0 +1,58 @@
+#define STATUS_EFFECT_DR_JEKYLL /datum/status_effect/dr_jekyll
+/obj/structure/toolabnormality/dr_jekyll
+	name = "dr jekyll's formula"
+	desc = "An innocent-looking bottle sitting on a table."
+	icon_state = "dr_jekyll"
+	var/list/users = list()
+
+/obj/structure/toolabnormality/dr_jekyll/attack_hand(mob/living/carbon/human/user)
+	..()
+	if(!do_after(user, 6))
```
Comment (dsmith328) [inline code/modules/mob/living/simple_animal/abnormality/_tools/teth/dr_jekyll.dm:10]:
```suggestion
	if(!do_after(user, 6, src))
```
Stops it from being spammed.

File: code/modules/paperwork/records/info/tools.dm [hunk 7]
```diff
@@ -105,3 +105,36 @@
 	- If the user’s HP drops below 50%, they will be healed. The user loses thread and experiences significant mental corrosion when this happens. <br>
 	- If the user’s HP drops below 50% when threads are too thin, they will collapse into pieces.
 	- The user can restore lost threads at the loom, sacrificing much of their own fortitude.<br> "}
+
+//Dr. Jekyll's Formula
+/obj/item/paper/fluff/info/tool/dr_jekyll
+	name = "F-09-148"
+	info = {"<h1><center>F-09-143</center></h1>	<br>
```
Comment (dsmith328) [inline code/modules/paperwork/records/info/tools.dm:112]: Wrong number?

File: code/modules/paperwork/records/info/tools.dm [hunk 8]
```diff
@@ -105,3 +105,36 @@
...
+	Risk Class : TETH	<br>
+	- The employee who drinks this will experience an increase in Fortitude.	<br>
+	- The abnormality can be drank until ones Fortitude reaches its limit, at which point it will always seem empty.	<br>
+	- For every time past the first drink, the employee will become vulnerable to RED damage.	<br>
+	- If the user takes SP damage, they begin to take equal amounts of self-inflicted HP damage. The user in question will claim "Mr. Hyde" to be responsible."}
+
+//Viva Voce
+/obj/item/paper/fluff/info/tool/vivavoce
+	name = "T-09-151" //Agoraphobia
+	info = {"<h1><center>O-01-19</center></h1>	<br>
```
Comment (dsmith328) [inline code/modules/paperwork/records/info/tools.dm:123]: Wrong number.

File: code/modules/paperwork/records/info/tools.dm [hunk 9]
```diff
@@ -105,3 +105,36 @@
...
+	Risk Class : HE	<br>
+	- If the employee listens to the abnormality, they will experience an increase in Temperance.	<br>
+	- If the employee continues to listen, the employee's Temperance will increase until it reaches its limit.	<br>
+	- Unless exposure is minimal, the employee will become increasingly vulnerable to BLACK damage.	<br>
+	- Exposure to this abnormality causes employees to become highly paranoid and careless when working with abnormalities, taking double damage."}
+
+//Theonite Slab
+/obj/item/paper/fluff/info/tool/theonite_slab
+	name = "O-09-144"
+	info = {"<h1><center>T-09-145</center></h1>	<br>
```
Comment (dsmith328) [inline code/modules/paperwork/records/info/tools.dm:134]: Wrong number!

**Review by dsmith328 — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/simple_animal/abnormality/_tools/he/theonite_slab.dm (hunk 4 above)
Comment (Coxswain-Navigator) [reply code/modules/mob/living/simple_animal/abnormality/_tools/he/theonite_slab.dm:10]: feature. Just like it's a feature for skin prophecy

**Review by dsmith328 — APPROVED** [review]

**Pull request merged**